Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -494,9 +498,11 @@ std::partial_ordering operator<=>(const AXTextMarker& marker1, const AXTextMarke
 
 bool AXTextMarkerRange::isConfinedTo(std::optional<AXID> objectID) const
 {
-    return m_start.objectID() == objectID
-        && m_end.objectID() == objectID
-        && LIKELY(m_start.treeID() == m_end.treeID());
+    if (m_start.objectID() != objectID || m_end.objectID() != objectID)
+        return false;
+    if (m_start.treeID() == m_end.treeID()) [[likely]]
```

Comment:
Does this likely even have an effect? Unless this entire function is inlined, which doesn't seem to be the case given the caller is in a different translation unit, this "likely" doesn't seem to have any impact?
Honestly, I'd just drop LIKELY here. We need to do all other comparisons anyway so it's not really buying us much.

Reply:
Ok.